qrenderdoc/Code/qrenderdoc.cpp

```diff
@@ -355,7 +380,10 @@ int main(int argc, char *argv[])
       }
 
       CaptureContext ctx(filename, remoteHost, remoteIdent, temp, config);
-
+      if(0 <= replayHostIndex)
```

Comment:
Prefer conventional `if(replayHostIndex >= 0)` order of comparisons, as it's easier to read for most people.